src/api/l_audio.c

```diff
@@ -305,20 +311,27 @@ int luaopen_lovr_audio(lua_State* L) {
 
   bool start = true;
   const char *spatializer = NULL;
+  int sampleRate = DEFAULT_SAMPLE_RATE;
```

Comment:
I'd rather move the 48000 here, since no one else uses the define.

Reply:
Made the change but—
I usually think it's better, rather than putting a magic number in code, to put the magic number in a constant and use the constant. What's concerning to me here specifically is this means the default will be set in an l_ file— if someone hypothetically were using the "C part" of the Lovr API standalone, they'd have no sense of what the default would be.

Reply:
Opps, I missed this comment, but yeah that's pretty reasonable.